test/scenarios-periodics/el92-src@olm.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Sourced from cleanup_scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart.ks.template rhel-9.2-microshift-source
+    launch_vm host1
+}
+
+scenario_remove_vms() {
+    remove_vm host1
+}
+
+scenario_run_tests() {
+    run_tests host1 suites/optional/olm.robot
```

Comment:
Would you like to rename the test to `optional` and run it on the whole directory?

Reply:
done